.github/workflows/eval.yml

```
@@ -165,7 +165,7 @@ jobs:
           runId=$(jq .id <<< "$run")
           conclusion=$(jq -r .conclusion <<< "$run")
 
-          while [[ "$conclusion" == null ]]; do
+          while [[ "$conclusion" == null && "$conclusion" != "" ]]; do
```

this looks slightly odd -- perhaps I'm not understanding something but maybe it should be:
```suggestion
          while [[ "$conclusion" == null || "$conclusion" == "" ]]; do
```
?

---

Oh damn you're right! Embarassing haha

---

I thought this was so simple it doesn't even need testing, and still I messed up :sweat_smile: